packages/react-ui/src/components/molecules/UploadField/StyledUploadField.js

```diff
@@ -23,7 +23,7 @@ const StyledUploadField = styled(TextField, {
     }
   },
   '& .MuiFormLabel-root': {
-    cursor: 'pointer',
+    cursor: cursor,
 
     '&.Mui-disabled': {
       pointerEvents: 'none'
```

Comment:
Really wondering if whole `...(focused && {` section below is needed.

Looks like this is really reimplementation of native behavior of TextField which ... already has `focused` boolean available ...

Reply:
I double-checked and is not needed, so removed, thanks!